lib/addons/p5.dom.js

```diff
@@ -58,12 +58,23 @@
    * </code></div>
    *
    */
-  p5.prototype.select = function (e) {
+  p5.prototype.select = function (e, p) {
     var res;
     var str;
+    var container = document;
+    if (typeof p === 'string'){
+      if(p[0] !== '#'){
+        container = document.getElementById(p);
```

Comment:
we should probably check that the result is not null. if it is, we should keep container = document, rather than setting it equal to the result of the search.